content/post/podcast/episode-2-2-talk-to-shardingsphere.md

```diff
@@ -52,8 +52,9 @@ authors:
 
 ## 请收听
 
-
-{{< ximalaya id="304115934" align="center" type="red" >}}
+```html
+<iframe height="230" width="260" src="https://www.ximalaya.com/thirdparty/player/sound/player.html?id=308619842&type=red" frameborder=0 allowfullscreen></iframe>
```

Comment:
please using the customer defined  xiamalaya tag.

Reply:
ok, I just change id number . Not all iframe.